util/weeklog.js

```diff
@@ -57,12 +63,11 @@ function parseLog (buffer, categoryList) {
         }
       }
 
-    } else {
-
-      log.push(entry);
-
+      continue;
     }
 
+    log.push(entry);
+
```

Comment:
This refactor makes no sense? Isn't the previous version much simpler to read here?